packages/expo-cli/src/commands/utils/PublishUtils.ts

```diff
@@ -63,6 +63,21 @@ export type PublicationDetail = {
 
 const VERSION = 2;
 
+async function getSlugAsync({
+  projectRoot,
+  exp,
+}: {
+  projectRoot: string;
+  exp: Pick<ExpoConfig, 'slug'>;
+}): Promise<string> {
+  if (exp.slug) {
```

Comment:
This check (and the whole function?) now seems redundant, because at some point we made it so that a `slug` is always returned from `getConfig`:
https://github.com/expo/expo-cli/blob/4aef8157645bed019ecc65d3ec216e59fedf6337/packages/config-types/src/index.ts#L17-L20

Reply:
nice